### PR TITLE
Preternis are tanks now

### PR DESCRIPTION
**Pull request by Sniblet**
# Document the changes in your pull request

Hi it's #15005 again (not exactly but same idea)

The [new lore](https://wiki.yogstation.net/wiki/Preternis#History) says that Preternis were made for working with hazardous environments including fusion engines, so continuing to treat them like they're made of overweight tin shavings isss kinda silly?

Rather than being both more fragile _and_ slower than humans and IPCs (what were you thinking), preternis are now physically tougher aaand even slower. They have innate armor like a lower-end golem, and resist shock almost as well as an ethereal. They're naturally just slightly faster than plantpeople (0.3 to plants' 0.33).

I didn't take the EMP resistance thing from 15005 because I don't understand why they'd have that and every nonhuman species must have at least two major ways to make players regret playing as them

# Wiki Documentation

Update "Health Management"

# Changelog

:cl: 
tweak: Preternis are even slower, but are significantly tougher against all damage types (except heat) than other species
/:cl:

### CODE REVIEW

**Comment by KillerOrcaCora** [comment]
"two major ways to make players regret playing as them" idk who made this rule but ok, races have downsides and perks to make them unique and not all the same, not so much because this is some kind of RPG pick your class table top, it's a rp game. 

I think this is boarding from being rp and more of a pick your class, and this whole thing of trying to slow them down so much is really gonna do the total opposite of what you want and make it played by absolutely no one. 

TL;DR You and skrem are both off the deep end on this one

**Comment by Rdrazga** [comment]
The 9 punch damage seems like a horribly bad idea and straight brute and burn mods are VERY powerful since there is no way to get damage through those mods, I understand the ‘lore reasoning’ but even with slowdown this is a bit much of you are not trying to make preternis the meta race

**Comment by Sniblet** [comment]
- The 9 punch stun threshold/damage has been in for months now and does not pertain to this PR
- Movement speed is more important than damage reduction for combat because not getting hit is how you avoid stuns and damage slowdown
- The "two major ways" thing was a joke because notice how podpeople and felinids both have multiple weird ways to grief them
- I don't understand the rest of what Orca is talking about

I was inspired to do this by the fact that preternis are, at this time, mechanically, a redundant species, because the only things they get that IPCs don't have better are night vision and punch force, which is not worth a minimum 25% greater weakness to all damage and .1 slowdown

**Comment by Skrem7** [comment]
if they aren't immune to emps you could probably give them more MS, IPCs get to have roundstart spacewalk bcuz they don't take space damage and also being piss easy to revive

I can look at the values themselves when I'm not busy as well as invoke more of my reasoning for the EMP thing

**Comment by Rdrazga** [comment]
> * The 9 punch damage has been in for months now and does not pertain to this PR
> * They're preternis not plasmamen
> * Movement speed is more important than damage reduction for combat because not getting hit is how you avoid stuns and damage slowdown
> * The "two major ways" thing was a joke but notice how podpeople and felinids both have multiple weird ways to grief them
> * I don't understand the rest of what Orca is talking about
> 
> I was inspired to do this by the fact that preternis are, at this time, mechanically, a redundant species, because the only things they get that IPCs don't have better are night vision and punch force, which is not worth a minimum 25% greater weakness to all damage and .1 slowdown

Yes I fixed saying they were plasmeme I misread, brute and burn mods can be stacked even in preternis to be around .6 iirc (and is still un-counterable unlike armor) and you can get a 5 damage flat reduction added to this before armor. Slowdown does not make up for this as there are a few ways to deal with slowdown and a good amount of situations where slowdown doesn’t matter. Reguardless of who did punch damage it is a bad idea since it stacks with martial art damage and makes gloves of the North Star really powerful.

The only other race that has stats comparable to this are a Draconiod and that is not round start

**Comment by Mqiib** [comment]
> The only other race that has stats comparable to this are a Draconiod and that is not round start


Draconids are a _horrible_ race to compare other species to because they are supposed to be an explicitly better version of a roundstart race, and an uncommon one at that.

**Comment by Skrem7** [comment]
Idea for EMP was that cameras get EMP resistance if plasma is installed, and considering the instability of some reactors that they might've been working in (think singulo emp), preterni plating would've been infused with plasma. Plasteel itself doesn't block EMP's though, so it's def something I wouldn't want to repeat. If I ever get around to making preterni ops, then those preterni will be EMP-immune. Fun-fun.

I would tick burn mod up to 0.9 and speedmod to 0.3. High heat mod is rough, and they don't just naturally regen like plant people do, nor can they really benefit from obnoxious drug mixes.

**Comment by Rdrazga** [comment]
> > The only other race that has stats comparable to this are a Draconiod and that is not round start
> 
> Draconids are a _horrible_ race to compare other species to because they are supposed to be an explicitly better version of a roundstart race, and an uncommon one at that.

The reason I am comparing these to a non roundstart race is because stat wise they have a LOT in common with draconoid, and no other round start race gets damage reduction to the extent that this pr is giving. This is on top of the preternis immunities they already had that are not being touched.

**Comment by Sniblet** [comment]
> no other round start race gets damage reduction to the extent that this pr is giving.

That's the point. If I only gave preternis things that other species already had, I would be making a boring change and doing nothing for their redundancy issue. How common damage mods are doesn't have any bearing here.
What _is_ your point - that there's no way around them? There's also no way around the fact that IPCs can't be poisoned, wounded, or suffocated, that preternis also can't be wounded, that ethereals run at 110% speed, that podpeople take half damage from romerol tumors, that plasmamen always have internals on... do you take issue with all of these?

**Comment by Rdrazga** [comment]
> > no other round start race gets damage reduction to the extent that this pr is giving.
> 
> That's the point. If I only gave preternis things that other species already had, I would be making a boring change and doing nothing for their redundancy issue. How common damage mods are doesn't have any bearing here. What _is_ your point - that there's no way around them? There's also no way around the fact that IPCs can't be poisoned, that ethereals run at 110% speed, that podpeople take half damage from romerol tumors, that polysmorphs are unusually difficult to wound... do you take issue with all of these?

All of these are races which have a very easily accessible damage type that a smart crew member can use, preternis already gets a ton of immunities and you are also adding damage reduction that CANNOT be countered, with a large amount of weapons especially AP ones being balanced around killing someone in x hits. At least make it 20 armor instead of .8 damage mod if you are dead set on this.

**Comment by Sniblet** [comment]
When you say "ton of immunities," are you referring to anything besides radiation and wounds?
20 armor is done.

**Comment by Cartlord** [comment]
Making them physically tough but slow is pretty pointless, because you're making them better at combat but also completely useless for it because movement speed is king - and that same move-speed debuff makes them annoying to play as a non-combat job, because those have to do a not-insignificant amount of walking anyway (e.g. Botanists between trays, Miners everywhere, Sci all around the station if they can be bothered to upgrade machines), so playing a Preternis in any of those jobs will just _suck_.
Ask yourself: for what reason would someone play Preternis? What situation will make you glad you're playing as one? What sick meme can you pull off as one that you can't do as another race, like how plasmemes can take off their suits & smoke in full-plasma rooms or podmen can walk off gunfire?
I believe that, on the grounds of almost every Preternis ability being already available to another race, we should either axe them entirely or re-balance all races, as I think we already have enough races that most styles of play are covered.

**Comment by Moggram** [comment]
I like the ideal that preternis are not a "combat race". You have to be more like a ninja to get a kill. If the damage mod is too high. then stop playing as one of us or get robust.

**Comment by Skrem7** [comment]
> I like the ideal that preternis are not a "combat race". You have to be more like a ninja to get a kill. If the damage mod is too high. then stop playing as one of us or get robust. 

If you do not care about combat then why do you not play IPC. They're better in every way out of combat. And also in-combat. If you only care about flavor then I'm not sure why you have grievances with this PR. Why does metal skin around flesh make you take more damage? It's nonsensical.

This PR seeks to give preternis a new identity and I'm sorry that it's being made not subpar IPC and more an adjacent species that's different in its strengths.

**Comment by Sniblet** [comment]
I’m not sure how anyone thought I was trying to make preternis a “combat race.”
The idea here is to make them very sturdy and survivable in every context except combat. Like I said at the start, movement is how you survive a fight. New preternis aren’t great at moving, but they are good at spacewalks, walking through fire (sort of), taking collateral gunshots, and then they self-repair. They would be a perfectly designed species for a world without conflict - and, that’s what they _are_ in the new lore.
Ultimately, they’re like podpeople, sort of. Podpeople who can take a laser shot without dissolving. Or current preternis who can take any kind of shot without dissolving. If you try to kill one it’ll be harder than killing a podperson, but bring enough gun, or an EMP, or a disabler, and it’s over - if they get away from you somehow, the next time you meet they’ll be at full health.

**Comment by Cartlord** [comment]
So you're making them like podpeople - another slow race that can take minor hits from simplemobs or the environment and walk them off.
Then remove podpeople, because preternis fill the same functions and _redundancy is bad._

**Comment by Moggram** [comment]
>If you do not care about combat then why do you not play IPC. 

Because IPC are heresy in the eyes of the **Omnissiah** and I ain't no dirty Tau player.

>If you only care about flavor then I'm not sure why you have grievances with this PR.

Who says I have to use the wiki lore. My guy was human who want overboard getting robot parts. As they are now (and have been) that works outs nicely.

>Why does metal skin around flesh make you take more damage? It's nonsensical.

Cheap mass produced parts that are easy to replace. If my arm gets blow off,  I can put it back on by myself. Also this PR say it wants to make us tankly, but didn't do anything to the lower limb heath.

>This PR seeks to give preternis a new identity

Ya, they are called IPC.

>I'm sorry that it's being made not subpar IPC and more an adjacent species that's different in its strengths.

Stuff can be under balanced and that is ok.

**Comment by Skrem7** [comment]
> Who says I have to use the wiki lore. My guy was human who want overboard getting robot parts. As they are now (and have been) that works outs nicely.

I agree, we should dev around your non-written headcanon that everyone else accepts.

**Comment by Moggram** [comment]
> > Who says I have to use the wiki lore. My guy was human who want overboard getting robot parts. As they are now (and have been) that works outs nicely.
> 
> I agree, we should dev around your non-written headcanon that everyone else accepts.

Its a Role Playing game and that is how I am role playing it. By the way that was the old wiki lore... not mine.

**Comment by Skrem7** [comment]
> > > Who says I have to use the wiki lore. My guy was human who want overboard getting robot parts. As they are now (and have been) that works outs nicely.
> > 
> > 
> > I agree, we should dev around your non-written headcanon that everyone else accepts.
> 
> Its a Role Playing game and that is how I am role playing it. By the way that was the old wiki lore... not mine.

https://wiki.yogstation.net/index.php?title=Preternis&oldid=38189

I would go on about how roleplay fundamentally requires agreement between a player, other players, and the fictional reality they are acting out. I don't care enough to.

**Comment by Moggram** [comment]
>https://wiki.yogstation.net/index.php?title=Preternis&oldid=38189

Nce [cheery picking](https://wiki.yogstation.net/index.php?title=Preternis&oldid=24195) there.

**Comment by Skrem7** [comment]
> > https://wiki.yogstation.net/index.php?title=Preternis&oldid=38189
> 
> Nce [cheery picking](https://wiki.yogstation.net/index.php?title=Preternis&oldid=24195) there.

Older version that existed before the previous lore team. Previous lore team scrapped the lore entirely and left nothing in its wake, seen here https://wiki.yogstation.net/index.php?title=Preternis&oldid=32595

Citing a page that nobody will see because for a whole 2 years the lore on the page was left as "TBD" is laughable. What matters is what has been presented. For two years that presentation was nothing.

**Comment by Sniblet** [comment]
Psychotic take saying that preternis must be left mechanically redundant and equally insane to say that they're functionally interchangeable with podpeople just because I used them as a comparison
I don't know why I argue for my PRs, seems like everybody just slams their faces on their keyboards and then reorganizes that into the nearest theoretically cogent argument against, until Jamie comes in and agrees with whichever side posted more
(ily jamie but i truly dont understand you)

**Comment by Cartlord** [comment]
I've tried to offer coherent arguments, and you've either ignored them or blown them off. Quit playin' victim, it ain't doin' you any favors.

**Comment by Sniblet** [comment]
I'm blowing off the argument that a robotic species with self-repair and innate armor is interchangeable with a plant-based species that regenerates over time and takes greatly amplified damage from all sources, and at that, is _more_ interchangeable than that same robotic species when it also takes greatly amplified damage from all sources.
I don't think I should have to explain why.
I think you can understand my frustration if you try to.

**Comment by Moggram** [comment]
> Older version that existed before the previous lore team. Previous lore team scrapped the lore entirely and left nothing in its wake, seen here https://wiki.yogstation.net/index.php?title=Preternis&oldid=32595
> 
> Citing a page that nobody will see because for a whole 2 years the lore on the page was left as "TBD" is laughable. What matters is what has been presented. For two years that presentation was nothing.

Bro, that was the lore when I stated playing on yogstation. Now then do you have a rebuttal for my other grievances? (I like to move on)

**Comment by Cartlord** [comment]
I am saying that nobody gives a shit about the difference, because welders & cables aren't any harder to come by than _light_ due to us having public Tool Storage areas & (on some maps) YouTool vendors and AutoLathes; therefore, they are functionally the same, being "slow fucks who can easily heal out of combat at little-to-no cost".
In addition, while I do admit preternis suck right now, you fail to being up why this revision makes them _worth using_ - there seems to be nothing they can do that other races can't, besides walk through fire without dying **quite** as fast; it seems like all the other things they can do (spacewalk with a firesuit, heal out of combat without needing med's aid, self-surgery) can be done by other races (plasmamen & maybe I.P.C. units, podmen and I.P.C. units, I.P.C. units).
To repeat what I brought up earlier: Ask yourself: for what reason would someone play Preternis? What situation will make you glad you're playing as one? What sick meme can you pull off as one that you can't do as another race, like how plasmemes can take off their suits & smoke in full-plasma rooms or podmen can walk off gunshots because they have a flashlight with them?
I believe that, on the grounds of almost every Preternis ability being already available to another race, we should either axe them entirely or re-balance all races, as I think we already have enough races that most styles of play and cool gimmicks are covered.

**Comment by Sniblet** [comment]
Preternis' unique abilities are their night vision, feeding method, self-surgery, and low oxygen requirement. IPCs scratched out the last two of these. I do aim to make them desirable again - there is no other roundstart species that has innate armor.

This PR would make preternis, an already somewhat unique race, more distinct, by giving them a third trait that no other race has. You seem to be arguing that preternis should be removed because they are not radically distinct at this time.

I don't think races are justified by their ability to stand around in rooms full of pure plasma and essentially act as a circus act for the entire round. I don't think they need that much to be justified.

What's the harm in having a lot of races with overlapping abilities? It looks superficially more like a furry server? Is the ability to accommodate minutely different tastes in robotic races too snowflakey? Preternis has always been a fairly unpopular race pick. So is ethereal, so is moth, so is polysmorph, so is plasmaman, so is podperson, and so, now, is IPC. You'll only see one or two, rarely three of each in every round, if that. So should we go back to just humans and lizards? And once we've done that, should we make it so you can only play lizard if you're in the medical department, since that's where most lizard players go? I, for one, must agree; I love restricting player choice.

**Comment by Cartlord** [comment]
...Uh, I had an idea that might make them cooler.
Since they're meant to be like factory worker bots, what if they got a boost to task speed? It wouldn't make them better in combat, but would give them _something_ that's entirely their own, and make playing one in a noncombat role less of a pain in the ass.

**Comment by Sniblet** [comment]
Not necessarily factory workers, but engine workers. The distinction is important because they're meant to be sturdy.
I don't know how to code action speed into a race, but it's a good idea. Reduce but keep their armor and movement penalty, and give them action speed.

**Comment by Skrem7** [comment]
They're meant to be industrial workers in general. They can withstand the radiation of engine burns, yes, so they're mostly ideal for that, but I also envisioned them being the individuals for anything hands-on in a hazardous environment, maybe not high temperatures because it's not really feasible to have any organic able to withstand that.

This does not have to be fully reflected as lore is just an accessory to gameplay, but I would almost give them some acid resistance, too. Not sure what I would take away. It's largely unnecessary regardless; acid is intensely rare, and it burns through plasteel all the same. Even just a smidgen of armor will help with that damage, too.

**Comment by Sniblet** [comment]
I'm not sure how race armor works, but most armor values are written as armor=list(melee, bullet, etc, acid, etc) whereas every racial armor bonus just has armor = number. I assume this means that all armor types are 20, so this PR as-is would give them 20 acid armor.
This is me assuming.

**Comment by Cartlord** [comment]
so is there any actual plan to give them fast arms or is that just going to sit in the cool idea bin for another thousand years or so

**Comment by athena413** [comment]
preternis are fine. people point at it and go "oh they need a rework!!!!! my balance!!!!!!"

my brother in Christ we do not need the most diverse crew. this isnt fucking Wal Mart where you need to have exact 1:1 parity for race player count. humans and lizards can stay as the default with the other ones existing as an interesting option for people who want to play. 

they have complete and total radiation immunity, with the easiest ability to self repair. the slowdown is not something i have experience in yet.

they are able to withstand harsh environments easily and all they need to be totally immune to everything is an atmospherics firesuit or an atmos hardsuit. this trait is shared with plasmapeople, but unlike plasmapeople they can also self repair with wires and a welder. unlike IPCs they also can benefit from chems against toxin and oxy damage as well as organ damage in general. chem purging doesnt matter that much in this case. this shit is perfect for actual hazardous environments

so why isnt this the most broken shit in the game? i mean, why ever play plasmaman or IPC if preternis exists?

simple: ANYONE who wants to kill you, can.

traitors have access to EMP grenades, changelings can EMP on demand, and anyone else with braincells can get their hands on some uranium and water. its not hard. and once they hit you with that EMP, you fucking die. being slower than (most) other races puts this shit truly in the grave too

they dont need a change they dont need a distinction, because they already have one. they work as is and can stand out if you actually bother to look at what theyre good at instead of just looking at what theyre bad at and going "wow, this race sucks shit! this needs a rework!" because never needing to take off your hazard vest or hardsuit in engineering to put on a radsuit for the engine/burnchamber is enough of a convenience to make me and a good few others play the species.

preternis, in gameplay terms alone, is a specialist race for  players who want to hard focus into a specific department with as little interruption as possible. if doing this allows any antag to onebang them with one of the most easily accessible weapons they have at their disposal - thats fine, the convenience for science / engineering players outweighs that entirely, and playing as them in other roles can be fun, too (being able to literally nonstop chainsmoke cigars with no downside as preternis brings me immense joy)

seriously, play this shit as a station engineer, atmos tech, or as sci/robo. its great

also - with the amount of stuns in SS13, resistance doesnt matter. cutting speed is going to kill this species and wont make them good.

**Comment by Hopekz** [comment]
Should have kept this open there is so many walls of text feedback here that there is no way someone with merge access would read any of this before they flip a coin to decide if this should be merged
